### ISSUE TITLE
Display Issues with Win10 display scaling options

### ISSUE THREAD

**Issue by Yohoki** [BullyWiiPlaza/Universal-Pointer-Searcher#9]
The display kind of.... freaks out when using Windows 10's display scaling options... it's hard to explain, so picture maybe? It's simple enough to change the scaling back to 100% and fixes the issue, but everything looks so tiny, especially text T.T

Any idea how to keep this from happening? Buttons and things reappear if the mouse is scrolled over them, I guess because they update the colors on mouseover. But text doesn't really appear unless it's highlighted.
![image](https://user-images.githubusercontent.com/44595371/64093833-68f3d000-cd1f-11e9-961d-54b3b78aabf2.png)

**Comment by BullyWiiPlaza** [comment]
May I suggest you to reduce your PC's resolution if the text appears too small? It's standard text, though. I used 125% scaling and it was working normally. Can you maybe try something like this? https://stackoverflow.com/questions/23780564

**Comment by Yohoki** [comment]
it's only at 1080. I could def reduce it, but gaming...... I'll go through the steps here in a moment.

I've also noticed that it works just fine, even scaled, if I don't click the "Browse..." button when adding a dump. As soon as that dialog closes (or maybe opens? hard to tell) the app starts showing issues. But that only happens when I have it scaled to 125%.

**Comment by Yohoki** [comment]
found a solution:
In Java's install location find the Java.exe file
right click -> Properties -> Compatibility -> "Overide High DPI Settings" and use "System"

Repeat this for javaw.exe, javaws.exe and jp2launcher.exe

**Comment by BullyWiiPlaza** [comment]
Great, so this can be closed. :)

**Comment by Yohoki** [comment]
Some one else brought this issue up on GBATemp Forum, and I've learned a bit more about the issue since I posted here. Since some one else brought up an issue to you again, I thought I'd give you some extra info on what's happening.

It seems to be related to Swing. Swing claims to be DPI-Aware, so windows lets it do it's own thing, but it isn't doing it right (or isn't actually DPI-Aware) and it freaks out.

You'll have to do your own research on how to fix the problem if your interested, but at least we know what's causing it now. :)

https://superuser.com/questions/988379/how-do-i-run-java-apps-upscaled-on-a-high-dpi-display
